CMakeLists.txt

```diff
@@ -68,7 +73,15 @@ foreach(ITEM IN ITEMS ${SCHEMAS})
     # NOTE the artifacts are written back into the source repository (see the README)
     set(IN_FILE "${CMAKE_SOURCE_DIR}/templates/${ITEM}")
     set(OUT_FILE "${CMAKE_SOURCE_DIR}/schemas/${ITEM}")
-    if(ITEM MATCHES "\\.json$") # pretty print
+    if(ITEM MATCHES "account-ecosystem...schema.json$") # TODO: Make this the default handling for all JSON schemas
```

Comment:
For demonstration purposes, we are limiting the new python-based metadata processing to `account-ecosystem`-related schemas. This treatment would be applied to all schemas in a final implementation.